project/Versions.scala

```diff
@@ -10,6 +10,7 @@ object Versions {
   val circeYaml = "0.15.2"
   val helidon = "4.0.10"
   val sttp = "3.10.3"
+  val sttp4 = "4.0.0-M24"
```

Comment:
we should use RC1 ;)

Reply:
(I have used this version in the newest commits)